util/network/Cargo.toml

```diff
@@ -14,4 +14,5 @@ ethereum-types = "0.3"
 ethkey = { path = "../../ethkey" }
 ipnetwork = "0.12.6"
 rlp = { path = "../rlp" }
+libc = "0.2.7"
```

Comment:
I think it's fine to use `0.2` here – this code have quite unspecific version requirements.